stub/build.gradle

```diff
@@ -5,6 +5,7 @@ plugins {
 description = "gRPC: Stub"
 dependencies {
     compile project(':grpc-core')
+    testCompile project(':grpc-testing')
```

Comment:
Hmm, I suppose there isn't.  I think this is how core pulled it in.  I'll switch it.